Example/package.json

```diff
@@ -79,7 +79,6 @@
         "react-native-fast-image": "8.3.4",
         "react-native-gesture-handler": "1.10.3",
         "react-native-image-picker": "git+https://github.com/FactorBench/react-native-image-picker.git#develop",
-        "react-native-indicators": "git+https://github.com/tonlabs/react-native-indicators.git",
```

Comment:
Do you really want to delete it from Example, as I see it's a peerDep

Reply:
It's not a peer (as it's a pure JS module)